eternalcore-core/src/main/java/com/eternalcode/core/util/ReflectUtil.java

```diff
@@ -45,6 +46,7 @@ public static List<Class<?>> scanClasses(String packageToScan, ClassLoader class
             return loadedClasses;
         }
         catch (IOException | ClassNotFoundException exception) {
+    Sentry.captureException(exception);
```

Comment:
Trochę łopatologiczne Lucky jakoś globalnie kiedyś łapał błędy ale nw czy to tylko na wątki działa

Reply:
poprawiłem indentację

Reply:
Ale łopatologiczne jest używanie wszędzie `Sentry#captureException`, czy co masz na myśli? Musimy tak robić bo ja nie łapię wszystkich błędów celowo, np. tych argumentów z LC, bo przecież tam to jest wina stricte usera

Reply:
Wątek w Javie ma coś takiego jak uncatched handler tyle, że musi wywalić wątek aby to złapało to część case już ogarnie... warto pomyśleć nad tym